payjoin/src/uri/mod.rs

```diff
@@ -1,8 +1,10 @@
 use std::borrow::Cow;
 
 use bitcoin::address::NetworkChecked;
+use error::BadEndpointError;
```

Comment:
Just a note This'd let you use it without a a prefix elsewhere in the module.

```suggestion
pub(crate) use error::BadEndpointError;
```

But I don't actually think you want to do this. I think if you follow my later suggestions, `BadEndpointError` can stay referenced in the `url_ext` and `error` modules only since you don't need to do manual coercion in mod if you just propagate it to an `InternalPjParseError::BadEndpoint` variant.